taxcalc/validation/taxsim27/taxcalc.py

```diff
@@ -0,0 +1,78 @@
+import argparse
+import os
+import sys
+import shutil
+
+usage_str = 'python taxcalc.py LYY_FILENAME [--save] [--help]'
+
+parser = argparse.ArgumentParser(
+        prog='',
+        usage=usage_str,
+        description=('Call Tax-Calculator tc CLI reading input data from '
+                     'specified TAXSIM-27 input file and writing output '
+                     'in TAXSIM-27 output format to a file with the '
+                     'specified input file name plus the .out-taxcalc '
+                     'extension.'))
+
+parser.add_argument('LYY_FILENAME', 
+                        help=('L is a letter that is valid taxsim_input.py L input and '
+                              'YY is valid taxsim_input.py YEAR (20YY) input.'),
+                        default='')
+
+parser.add_argument('--save',
+                        help=('skips the deletion of itermediate files at the end of this script.'),
```

Comment:
It might be clearer to say something like:

> Save intermediate files.

or 

> Save files that were used to create the final data file.

Reply:
Agreed, went with your first suggestion